cli/lib/commands/whoami.js

```diff
@@ -8,19 +8,19 @@ const { whoAmI } = require('../utils');
 const whoamiOpts = figgy({
   registry: { default: 'https://registry.entropic.dev' },
   token: true,
-  log: { default: require('npmlog') }
+  log: true,
+  api: true
 });
 
 async function whoami(opts) {
   opts = whoamiOpts(opts);
 
   try {
-    const username = await whoAmI();
-
-    console.log(username);
+    const username = await whoAmI(opts);
+    opts.log.log(username);
```

Comment:
Agreed! I think it would be more idiomatic for it to read `opts.logger.log`, but that will require a larger PR because some parts of the code are currently using `opts.log.error`